packages/expo-av/src/ExponentAV.web.ts

```diff
@@ -164,10 +168,72 @@ export default {
 
   /* Recording */
   //   async setUnloadedCallbackForAndroidRecording() {},
-  async getAudioRecordingStatus() {},
-  async prepareAudioRecorder() {},
-  async startAudioRecording() {},
-  async pauseAudioRecording() {},
-  async stopAudioRecording() {},
-  async unloadAudioRecorder() {},
+  async getAudioRecordingStatus() {
+    return {
+      isRecording: _mediaRecorder && _mediaRecorder.state === 'recording',
+      isDoneRecording: false,
+      durationMillis: 2000,
+    };
+  },
+  async prepareAudioRecorder(options) {
+    if (!navigator.mediaDevices) {
+      throw new Error('No media devices available');
+    }
+
+    const stream = await navigator.mediaDevices.getUserMedia({ audio: true });
+
+    _mediaRecorder = new (window as any).MediaRecorder(
+      stream,
+      options?.web || RECORDING_OPTIONS_PRESET_HIGH_QUALITY.web
+    );
+
+    return { uri: 'file:///unsupported', status: await this.getAudioRecordingStatus() };
```

Comment:
```suggestion
    return { uri: '', status: await this.getAudioRecordingStatus() };
```

Reply:
Thanks for all the feedback! Working my way through it.

I return this file path so that the web recording passes the same tests, I'm happy to make this suggested change and I do believe it is better but it does mean conditionally changing some tests on web if you are happy for me to do that?

Reply:
Hey @EvanBacon, I appreciate you are super busy so I rolled the above change into a commit and if it's not ideal we can just as easily roll it back :) Thanks, Warren